soc_023_rw1_fragile_states_index/soc_023_rw1_fragile_states_index_processing.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import pandas as pd
+import glob
+import os
+import sys
+import dotenv
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+import util_carto
+from bs4 import BeautifulSoup
+import urllib
+import requests
+from zipfile import ZipFile
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
+dataset_name = 'soc_023_rw1_fragile_states_index' #check
+
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+logger.info('Executing script for dataset: ' + dataset_name)
+# create a new sub-directory within your specified dir called 'data'
+# within this directory, create files to store raw and processed data
+data_dir = util_files.prep_dirs(dataset_name)
+
+'''
+Download data and save to your data directory
+Excel files can be downloaded from:
+https://fragilestatesindex.org/excel/
+'''
+# Retrieve the html page from source
+req = urllib.request.Request("https://fragilestatesindex.org/excel/", headers={'User-Agent': 'Mozilla/5.0'})
+html_page = urllib.request.urlopen(req)
+soup = BeautifulSoup(html_page, "lxml")
+# Find all the links in html document
+links = []
+for link in soup.findAll('a'):
+    links.append(link.get('href'))
+# Drop duplicated links
+links = list(set(links))
+# Create list containing only links of xlsx files
+url_list = []
+for link in links:
+    if link.endswith(('.xlsx')):
+        url_list.append(link)
+# Create path to files
+raw_data_file = [os.path.join(data_dir,os.path.basename(url)) for url in url_list]
+# Download data from source
+for index,element in enumerate(url_list):
+    res = requests.get(element, headers={'User-Agent': 'Mozilla/5.0'})
+    res.raise_for_status()
+    # write the file to data directory
+    with open(raw_data_file[index], 'wb') as f:
+        f.write(res.content)
+
+# Go through data directory and append xlsx files into a list
+# Then concatenate the content of the list in a dataframe
+df_list = []
+for file in raw_data_file:
+    df = pd.read_excel(file)
+    df_list.append(df)
+df = pd.concat(df_list, ignore_index=True)
+
+# Create datetime column
+df['datetime'] = df['Year']
+# Changing dtype of year column
+df['Year'] = df['Year'].dt.year.astype(int)
```

Comment:
Since the 'Year' column already stores datetime objects, I don't think it's necessary to have another column storing the years as integers. Let me know if you think otherwise. 
```suggestion
```